EntAppProject/src/main/resources/templates/JBLogin.html

```diff
@@ -1,45 +1,102 @@
+	<link rel="stylesheet" type="text/css" href="../static/vendor/daterangepicker/daterangepicker.css">
+<!--===============================================================================================-->
+	<link rel="stylesheet" type="text/css" href="../static/css/JobBookLoginUtil.css">
+	<link rel="stylesheet" type="text/css" href="../static/css/JobBookLogin.css">
+<!--===============================================================================================-->
 </head>
 <body>
- 
-<h1><span>Job</span>Book</h1>
- 
-<h2>Welcome to JobBook</h2>
-<p></p>
- 
-<form action="action_page.php" method="post">
-<div class="container">
-        <label for="username"><b>Username</b></label>
-        <input type="text" placeholder="Username:" name="username" required>
-        <br>
-        <br>
-        <label for="userpassword"><b>Password</b></label>
-        <input type="password" placeholder="Password" name="userpassword" required>
-        <br>
-        <form method="get" action="/JBHomeScreen">
-        <button type="submit">Login</button>
-        </form>
-        <label>
-        
-        <input type="checkbox" checked="checked" name="remember"> Remember me?
-        </label>
-        </div>
-        <br>
-        <br>
-        </form>
-        <a href="/JBForgotPassword">Forgot Username/Password? Click Here.</a>
-<br>
-<br>
-<br>
-<br>
-        <a href="/JBCreateAccount">New to JobBook? Create an Account Here.</a>
- 
+	
+	<div class="limiter">
+		<div class="container-login100">
+			<div class="wrap-login100 p-l-55 p-r-55 p-t-65 p-b-50">
+				<form class="login100-form validate-form">
+					<span class="login100-form-title p-b-33">
+						Welcome to JobBook
+					</span>
+
+					<div class="wrap-input100 validate-input" data-validate = "Valid username is required: username123">
+						<input class="input100" type="text" name="username" placeholder="Username">
+						<span class="focus-input100-1"></span>
+						<span class="focus-input100-2"></span>
+					</div>
+
+					<div class="wrap-input100 rs1 validate-input" data-validate="Password is required">
+						<input class="input100" type="password" name="pass" placeholder="Password">
+						<span class="focus-input100-1"></span>
+						<span class="focus-input100-2"></span>
+					</div>
+
+					<div class="container-login100-form-btn m-t-20">
+						<button class="login100-form-btn">
+							Sign in
+						</button>
+					</div>
+
+					<div class="text-center p-t-45 p-b-4">
+						<span class="txt1">
+							Forgot
+						</span>
+
+						<a href="/JBForgotPassword" class="txt2 hov1">
+							Username / Password?
+						</a>
+					</div>
+
+					<div class="text-center">
+						<span class="txt1">
+							Create an account?
+						</span>
+
+						<a href="/JBCreateAccount" class="txt2 hov1">
+							Sign up
+						</a>
+					</div>
+				</form>
+			</div>
+		</div>
+	</div>
+	
+
+	
+<!--===============================================================================================-->

```

Comment:
These are the javascript files that are used to manage the pages actions. These files are all page-specific so nothing included in the files will negatively impact the rest of the application. They simply help improve the visual appeal of the Login Page and allow for cool features like animations. If the team decides that these changes are too resource heavy, I would recommend removing files from here first, and then removing stylesheets last.